user-interface/1005-VEST-vesting.md

```diff
@@ -1 +1,75 @@
-# Vesting
\ No newline at end of file
+# Vesting
+
+Some governance tokens may be held by a Vesting contract. This means that can be "owned" by an Ethereum key but not freely transferred until a vesting terms are complete.
+
+## list of tranches
+
+When looking to understand to overall vesting schedule for tokens, I...
+
+- **must** see a list of tranches <a name="1005-VEST-001" href="#1005-VEST-001">1005-VEST-001</a>
+- **should** see a visualization of the vesting schedule with a break down of the type of token holders (e.g. team, investors, community) <a name="1005-VEST-002" href="#1005-VEST-002">1005-VEST-002</a>
+
+For each tranche:
+
+- **must** see a tranche number <a name="1005-VEST-003" href="#1005-VEST-003">1005-VEST-003</a>
+- **could** see any annotation of what this tranche is about (e.g. community schedule A)
+- **must** see a sum of tokens in the tranche <a name="1005-VEST-005" href="#1005-VEST-005">1005-VEST-005</a>
+  - **must** see how many tokens in the tranche are locked <a name="1005-VEST-006" href="#1005-VEST-006">1005-VEST-006</a>
+  - **must** see how how many tokens in the tranche are redeemable <a name="1005-VEST-007" href="#1005-VEST-007">1005-VEST-007</a>
+- **must** see the vesting terms for each tranche (when unlocking stats and ends) <a name="1005-VEST-008" href="#1005-VEST-008">1005-VEST-008</a>
+
+... so I can understand how circulating supply could change over time.
+
+## Details of a tranche
+
+When looking into a specific tranche, I...
+
+- **must** see all the same details as the [list of tranches](#details-of-a-tranche) <a name="1005-VEST-012" href="#1005-VEST-012">1005-VEST-012</a>
+- **must** see a list of ethereum wallets with tokens in this tranche <a name="1005-VEST-013" href="#1005-VEST-013">1005-VEST-013</a>
```

Comment:
Do we have to have this? I'd really like to remove this requirement if at all possible. It's just such as massive pain.

Reply:
happy to remove...

Reply:
removed in https://github.com/vegaprotocol/specs/pull/1301/commits/093322aa3a24cf899bbe4a27dac3078dc43f55f5 but I'd like to sense check I understand correctly.
I would have thought that showing the same headline info for a tranche that you see in the trances list is easier on the individual tranche details page. but my not understanding is non-blocking

Reply:
resolved in https://github.com/vegaprotocol/specs/pull/1301/commits/db1d17adb01d0fa941d3fb4242d9463706e0edbb